### DISCUSSION TITLE
Feature Request: Permit to define what goes into a release zip/tar

### DISCUSSION THREAD

**Discussion by erinmez** [community/community#6444] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi there,

We use a mono repo where we have collaterals that are not source/build relevant. It would be great if we could use glob patterns to "select" what goes into a release. E.g.:
 `**/src/*, **/docs/*`

Best, Tanju

**Comment by MylesBorins** [comment, marked as answer]
hey @erinmez . At this time we are not going to focus on allowing changes to the archive format, but it does look like git supports some funcationality with .gitattributes for this.

If we do pick up work on improving the overall tarball / zipball auto archive service I'll make sure we discuss this and think through potential solutions.
